.github/workflows/publish-helm-charts.yaml

```diff
@@ -0,0 +1,37 @@
+name: Publish Helm charts
+
+on:
+  release:
+    types:
+      - published
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: false
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+        contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: main
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+
+      - name: Run chart publisher script
+        run: |
+          ./scripts/build/helm-publisher.sh \
+          ${{ join(github.event.release.assets.*.browser_download_url, ' ') }} \
+          ${{ github.event.release.tag_name }}
+        shell: bash
+
+      - uses: stefanzweifel/git-auto-commit-action@v4
```

Comment:
My question regarding this is that should we have a policy for what kind of actions to use in the repo? I'd be inclined to only use actions from verified publishers (or just the GitHub "official" actions) and enforce this in the repo settings. Thoughts @klihub @askervin @kad @fmuyassarov?

Reply:
Generally I agree about using from official or very well-known places. Do we have equivalent to this there?

Reply:
I have replaced the action-image with plain git commands.